docs/vtex-io/Reference/concepts/vtex-io-documentation-builders/vtex-io-documentation-store-builder.md

```diff
@@ -0,0 +1,113 @@
+---
+title: "Store builder"
+slug: "vtex-io-documentation-store-builder"
+excerpt: "Learn how to use the VTEX IO Store builder."
+hidden: false
+createdAt: "2024-05-02T09:33:00.000Z"
+updatedAt: "2024-05-02T09:33:00.000Z"
+category: "App Development"
+---
+
+The `store` builder enables the development of [Store Framework](https://developers.vtex.com/docs/guides/vtex-io-documentation-what-is-vtex-store-framework) storefronts, empowering the development of both [storefront components](LINK) and unique store themes. The main use cases of the `store` builder include:
```

Comment:
O link ficou pendente
Não é o ideal, mas por enquanto podemos deixar linkado para:
https://developers.vtex.com/docs/guides/vtex-io-documentation-1-developing-storefront-apps-using-react-and-vtex-io

Idealmente, seria uma doc específica de apps de front da parte de VTEX IO Apps